### PR TITLE
Add explanation how to push _config.yml changes

### PR DESCRIPTION
**Pull request by jonico**
* small shell snippet showing steps how to commit and push to your Hubble repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,12 @@ The **docs** component visualizes the collected data with [GitHub Pages](https:/
    git remote add ghe https://git.company.com/scm/hubble
    git push -u ghe master
    ```
-1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier.
-   Commit and push the change to your Hubble repository.
+1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier. Commit and push the change to your Hubble repository:
+   ```sh
+   git add docs/_config.yml
+   git commit -m "Adjusting dataURL to our own instance"
+   git push
```
Comment (larsxschneider) [inline README.md]: Thanks for the contribution! 👍 

At this point I think we should recommend the user to push to `ghe` remote, no?

**Comment by jonico** [comment]
@larsxschneider: because of the previous push -u, the tracking branch is already set to point to ghe, but we can explicitly add this remote to be on the safe side

Thoughts?

**Comment by larsxschneider** [comment]
You got me 😄  👍

File: README.md [hunk 2]
```diff
@@ -36,8 +36,12 @@ The **docs** component visualizes the collected data with [GitHub Pages](https:/
    git remote add ghe https://git.company.com/scm/hubble
    git push -u ghe master
    ```
-1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier.
-   Commit and push the change to your Hubble repository.
+1. Open [`docs/_config.yml`](docs/_config.yml) in your editor and set the `dataURL` that you noted earlier. Commit and push the change to your Hubble repository:
```
Comment (pluehne) [inline README.md]: Why did you merge the two lines into one? I believe diffs are easier to read with each sentence on a separate line.

**Pull request merged**